### ISSUE TITLE
Empty button JS rule missed errors

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#952]
Empty button misses content that is hidden with aria-hidden. Example code that should fail:
`<button><span aria-hidden="true">X</span></button>`

Also this should fail:
`<button><span style="display: none;">Menu</span><i class="fas fa-bars"></i></button>`

https://a11ycheckrules.wpenginepowered.com/wp-admin/post.php?post=1000&action=edit

**Comment by pattonwebz** [comment]
Not skipping buttons with aria-hidden was intentional here. Since buttons are still focusable with aria hidden, but screen readers see nothing when focus reaches them I think it's a violation of 4.1.2 at A level. https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html
